### PR TITLE
replace TimeLimitProcess with Commons Exec

### PR DESCRIPTION
**Pull request by markro49**
This replaces usage in src/systemtest/java/randoop/main/ProcessStatus.java; there is still one more to go: src/main/java/randoop/main/Minimize.java.

### CODE REVIEW

File: src/systemtest/java/randoop/main/ProcessStatus.java [hunk 1]
```diff
@@ -64,48 +66,59 @@ public String dump() {
...
-        new BufferedReader(new InputStreamReader(p.getInputStream(), UTF_8))) {
-      String line = rdr.readLine();
-      while (line != null) {
-        outputLines.add(line);
-        line = rdr.readLine();
+    try {
+      String buf = outStream.toString();
+      // Don't create a list with a single, empty element.
+      if (buf.length() > 0) {
+        outputLines = Arrays.asList(buf.split(lineSep));
```
Comment (mernst) [inline src/systemtest/java/randoop/main/ProcessStatus.java]: If the only use of outputLines is to print it, why split into lines at all?

**Comment by markro49** [comment]
About half the time we just print outputLines and about half the time we parse them.  So we could simplify the former but would need to add something like Arrays.asList(outputLines.split(lineSep)) to each of the latter.  Your call.

**Comment by mernst** [comment]
Would removing the call here entail changes anywhere else?
I was really asking about this particular occurrence.  I'm fine with splitting into lines when parsing, but I wouldn't split into lines everywhere just to make all the code textually consistent.  (Doing so might confuse programmers who wonder why, and incidentally it's unnecessary computation.)

**Comment by markro49** [comment]
I assumed you were suggesting changing the definition of outputLines from List<String> to just String.  In which case all clients need to change, some get simpler, some get slightly more complex.

**Comment by mernst** [comment]
I think we may be talking at cross purposes.
`outputLines` is a local variable in Randoop.  There are no clients of it.

**Comment by markro49** [comment]
but the local outputLines is passed to the ProcessStatus constructor to become the outputLines member of the returned ProcessStatus object.

**Comment by mernst** [comment]
Thanks for the clarification.  I thought I had looked and didn't see any other use, but I obviously overlooked that.  Sorry about that!
Either way is fine with me, and in particular the current way is fine.

**Pull request merged**